### PR TITLE
docs:  add 4.9.4 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 4.9.4

- 🐞 Fix Menu delayed hover color transition in Chrome. [#28372](https://github.com/ant-design/ant-design/pull/28372)
- 🐞 Fix the problem that the node connection line of Tree is set to relative position. [#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
- 💄 Fix Table fixed column `z-index` which makes Dropdown not showing on top. [#28346](https://github.com/ant-design/ant-design/pull/28346)
- TypeScript
  - 🤖 Fix `message.loading()` return type. [#28362](https://github.com/ant-design/ant-design/pull/28362)

-------

- 🐞 修复 Menu 在 Chrome 下 hover 色延迟变化的问题。[#28372](https://github.com/ant-design/ant-design/pull/28372)
- 🐞 修复 Tree 的节点连接线设定为相对位置的问题。[#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
- 💄 修复 Table 固定列 `z-index` 样式让 Dropdown 无法展示在最顶层的问题。[#28346](https://github.com/ant-design/ant-design/pull/28346)
- TypeScript
  - 🤖 修复 `message.loading()` 返回函数的定义。[#28362](https://github.com/ant-design/ant-design/pull/28362)

### CODE REVIEW

**Comment by afc163** [comment]
4.9.4

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 4.9.4
+
+- 🐞 Fix Menu delayed hover color transition in Chrome. [#28372](https://github.com/ant-design/ant-design/pull/28372)
+- 🐞 Fix the problem that the node connection line of Tree is set to relative position.ion. [#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
```
Comment (xrkffgg) [inline CHANGELOG.en-US.md:21]: position.ion.  ?

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 4.9.4
+
+- 🐞 Fix Menu delayed hover color transition in Chrome. [#28372](https://github.com/ant-design/ant-design/pull/28372)
+- 🐞 Fix the problem that the node connection line of Tree is set to relative position. [#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
```
Comment (afc163) [inline CHANGELOG.en-US.md:21]:
```suggestion
- 🐞 Fix Tree node connection line position problem. [#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 4.9.4
+
+- 🐞 修复 Menu 在 Chrome 下 hover 色延迟变化的问题。[#28372](https://github.com/ant-design/ant-design/pull/28372)
+- 🐞 修复 Tree 的节点连接线设定为相对位置的问题。[#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:21]:
```suggestion
- 🐞 修复 Tree 的节点连接线在滚动时位置偏移的问题。[#28354](https://github.com/ant-design/ant-design/pull/28354) [@maksnester](https://github.com/maksnester)
```

**Pull request merged**